base/shell/explorer/syspager.cpp

```diff
@@ -21,12 +21,6 @@
 
 #include "precomp.h"
 
-struct InternalIconData : NOTIFYICONDATA
-{
-    // Must keep a separate copy since the original is unioned with uTimeout.
-    UINT uVersionCopy;
-};
-
```

Comment:
Why isn't this needed anymore?

Reply:
No, it is moved to shell/explorer/precomp.h